Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM python:3.7-slim
+ARG PYTHON_VERSION=3.7
```

Comment:
Maybe remove the default value? It doesn't help, and it would be preferable to err if the `--build-arg` isn't passed.

Reply:
Otherwise Travis configs look good, good job @tkaemming.

Reply:
> Maybe remove the default value? It doesn't help, and it would be preferable to err if the `--build-arg` isn't passed.

I left this in here mostly for the benefit of the other build processes beyond Travis (most of which I admittedly don't have a very good grasp on), and this seemed like a good spot to define what we run in production by default to minimize the total number of changes.